op-bindings/bindgen/remote_handlers.go

```diff
@@ -0,0 +1,282 @@
+
+	"github.com/ethereum-optimism/optimism/op-bindings/etherscan"
+)
+
+type contractData struct {
+	abi          string
+	deployedBin  string
+	deploymentTx etherscan.TxInfo
+}
+
+func (generator *bindGenGeneratorRemote) standardHandler(contractMetadata *remoteContractMetadata) error {
+	fetchedData, err := generator.fetchContractData(contractMetadata.Verified, "eth", contractMetadata.Deployments["eth"])
+	if err != nil {
+		return err
+	}
+
+	contractMetadata.DeployedBin = fetchedData.deployedBin
+
+	// If ABI was explicitly provided by config, don't overwrite
+	if contractMetadata.Abi == "" {
+		contractMetadata.Abi = fetchedData.abi
+	} else if fetchedData.abi != "" && contractMetadata.Abi != fetchedData.abi {
+		generator.logger.Warn("The given contract ABI differs from what was fetched from Etherscan", "contract", contractMetadata.Name)
+		generator.logger.Debug("ABIs", "given", contractMetadata.Abi, "fetched", fetchedData.abi)
+	}
+
+	if contractMetadata.InitBin, err = generator.removeDeploymentSalt(fetchedData.deploymentTx.Input, contractMetadata.DeploymentSalt); err != nil {
+		return err
+	}
+
+	// We're not comparing the bytecode for Create2Deployer with deployment on OP,
+	// because we're predeploying a modified version of Create2Deployer that has not yet been
+	// deployed to OP.
+	// For context: https://github.com/ethereum-optimism/op-geth/pull/126
+	if contractMetadata.Name != "Create2Deployer" {
+		if err := generator.compareBytecodeWithOp(contractMetadata, true, true); err != nil {
+			return fmt.Errorf("error comparing contract bytecode for %s: %w", contractMetadata.Name, err)
+		}
+	}
+
+	return generator.writeAllOutputs(contractMetadata, remoteContractMetadataTemplate)
+}
+
+func (generator *bindGenGeneratorRemote) multiSendHandler(contractMetadata *remoteContractMetadata) error {
+	// MultiSend has an immutable that resolves to this(address).
+	// Because we're predeploying MultiSend to the same address as on OP,
+	// we can use the deployed bytecode directly for the predeploy
+	fetchedData, err := generator.fetchContractData(contractMetadata.Verified, "op", contractMetadata.Deployments["op"])
+	if err != nil {
+		return err
+	}
+
+	contractMetadata.Abi = fetchedData.abi
+	contractMetadata.DeployedBin = fetchedData.deployedBin
+	if contractMetadata.InitBin, err = generator.removeDeploymentSalt(fetchedData.deploymentTx.Input, contractMetadata.DeploymentSalt); err != nil {
+		return err
+	}
+
+	return generator.writeAllOutputs(contractMetadata, remoteContractMetadataTemplate)
+}
+
+func (generator *bindGenGeneratorRemote) senderCreatorHandler(contractMetadata *remoteContractMetadata) error {
+	var err error
+	contractMetadata.DeployedBin, err = generator.contractDataClients["eth"].FetchDeployedBytecode(contractMetadata.Deployments["eth"])
+	if err != nil {
+		return fmt.Errorf("error fetching deployed bytecode: %w", err)
+	}
+
+	if err := generator.compareBytecodeWithOp(contractMetadata, false, true); err != nil {
```

Comment:
description here why deployment isn't being compared?

Reply:
I see the block comment in the previous file but just to remain consistent with the comments in the other handlers